proposals/lip-0051.md

```diff
@@ -135,8 +135,8 @@ The following constants are used throughout the document:
 | **Token Store Constants** | | |
 | `SUBSTORE_PREFIX_USER` | bytes | 0x 00 00 |
 | `SUBSTORE_PREFIX_SUPPLY` | bytes | 0x 80 00 |
-| `SUBSTORE_PREFIX_ESCROW` | bytes | 0x c0 00 |
-| `SUBSTORE_PREFIX_SUPPORTED_TOKENS` | bytes | 0x e0 00 |
+| `SUBSTORE_PREFIX_ESCROW` | bytes | 0x 40 00 |
```

Comment:
Very minor, but since we're at this I would remove the white space to align with the other LIPs

Reply:
Updated. I also removed white spaces in other parts to align within the LIP.

Reply:
This substore starts with a `0` and hence will increase the length for recoveries from the user substore by one extra hash. This was the original reason for the choice of substore prefixes and I would maintain it unless it introduces too much complexity on the implementation

Reply:
Currently substoreprefix is [computed in sdk](https://github.com/LiskHQ/lisk-sdk/blob/98c3d9c588b56845e46eb8a209985a92bf478c67/framework/src/modules/base_store.ts#L35) by passing the index and index for these substores are 0, 1, 2, 3 as mentioned [here](https://github.com/LiskHQ/lisk-sdk/blob/cc8cbba91ff5baacae087657a9e010c6f31f32a4/framework/src/modules/token/module.ts#L96).  Maybe @shuse2 can better answer here about the implementation change?

Reply:
@ricott1 Should be update token module to have subStore starting from `index 1` on SDK?

Reply:
@ishantiw I think you did the same for the [Interoperability module ](https://github.com/LiskHQ/lisk-sdk/blob/98c3d9c588b56845e46eb8a209985a92bf478c67/framework/src/modules/interoperability/base_interoperability_module.ts#L54) by choosing the substore index that yields the substore prefix that is defined in the LIP.

For Token, I think 
```
computeSubstorePrefix(3) == <Buffer c0 00>
computeSubstorePrefix(7) == <Buffer e0 00>
```
We could then leave the Token module LIP as is and only update the implementation.

Reply:
@ishantiw We keep the indexes 0,1, 2, 3 according to the sdk implementation.

Reply:
okay so that means no changes on token module SDK side 👍